From 07d9b7c07b021b408665046e2f11558e3ca9f83d Mon Sep 17 00:00:00 2001
From: Jonas Karlman <jonas@kwiboo.se>
Date: Sun, 14 Apr 2019 11:26:42 +0000
Subject: [PATCH 07/10] FIXUP: rename av_buffer_pool_reclaim to
 av_buffer_pool_flush

---
 doc/APIchanges | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/APIchanges b/doc/APIchanges
index cf426e2..5953ee8 100644
--- a/doc/APIchanges
+++ b/doc/APIchanges
@@ -15,6 +15,9 @@ libavutil:     2017-10-21
 
 API changes, most recent first:
 
+2019-XX-XX - xxxxxxxxxx - lavu 56.XX.100 - buffer.h
+  Add av_buffer_pool_flush().
+
 2019-05-15 - b79b29ddb1 - lavu 56.28.100 - tx.h
   Add av_tx_init(), av_tx_uninit() and related definitions.
 
-- 
2.16.4

